docs/terms/cli.md

```diff
@@ -1,7 +1,94 @@
 ---
 title: CLI (Command Line Interface)
-id: cli
-hoverText: Command Line Interface
 ---
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '/docs/images/universal_map/_um_setup_header.mdx'
+import ConnectHeader from '/docs/images/universal_map/_um_connect_header.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='active' connect='active' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
+
+CLI stands for Command Line Interface.
+
+### Features and promises
+
+The CLI provides useful convenience functions covering all the steps of working with Great Expectations.  CLI commands consist of a noun indicating what you want to operate on, and a verb indicating the operation to perform.  All CLI commands have help documentation that can be accessed by including the `--help` option.
+
```

Comment:
Note that you can also get all these options by just running `great_expectations` without any args.

Reply:
I added that as a way of getting a list of the available commands.